### PR TITLE
Docker scripts for an iptables-enabled container which can run agent

### PR DESCRIPTION
**Pull request by a-martynovich**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +1,20 @@
...
+FROM debian
 WORKDIR /usr/src/app
-ENV PYTHONUNBUFFERED=1
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends build-essential libssl-dev libffi-dev libltdl-dev curl && \
-    apt-get clean
+    apt-get install -y build-essential libssl-dev libffi-dev libltdl-dev openssh-server git net-tools \
+        iptables inetutils-ping wget curl pkg-config libsystemd-dev nmap python3-iptables python3-setuptools python3-pip && \
+    service ssh start && \
```
Comment (vpetersson) [inline Dockerfile]: SSH should never be enabled in a docker container. Use `docker exec` instead.

File: Dockerfile [hunk 2]
```diff
@@ -1,34 +1,20 @@
...
-    apt-get clean
-
-RUN git clone https://github.com/square/ghostunnel.git $GOPATH/src/github.com/square/ghostunnel && \
-    cd  $GOPATH/src/github.com/square/ghostunnel && \
-    git checkout -b v1.3.0 && \
-    make
-
-## Runtime container
-FROM python:3.7-slim-stretch
+# docker run --net=host --cap-add=NET_ADMIN --cap-add=SYS_ADMIN  -v /sys/fs/cgroup:/sys/fs/cgroup:ro --security-opt seccomp:unconfined -v ~/Documents/GreatFruit/WoTT:/usr/src/app -Pit mydebian
```
Comment (vpetersson) [inline Dockerfile]: Let's move this to a bash script instead.

File: Dockerfile [hunk 3]
```diff
@@ -1,34 +1,20 @@
...
+FROM debian
```
Comment (vpetersson) [inline Dockerfile]: Let's pin this better to say `debian:stretch`

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  main:
+    command: bash
```
Comment (vpetersson) [inline docker-compose.yml]: It would make more sense if we launched the wott-agent in foreground mode (i know we don't have this feature yet).

Comment (a-martynovich) [reply docker-compose.yml]: @vpetersson We do have this feature, but this container is for running agent manually for development, not for keeping it running all the time.

File: Dockerfile (hunk 2 above)
Comment (a-martynovich) [reply Dockerfile]: I'll remove this, this command is exactly what docker-compose does.

Comment (vpetersson) [reply Dockerfile]: Ah right.

File: docker-compose.yml (hunk 4 above)
Comment (vpetersson) [reply docker-compose.yml]: Alright - fair enough.

**Pull request merged**